earth2studio/models/dx/solarradiation_afno.py

```diff
@@ -0,0 +1,363 @@
+# SPDX-FileCopyrightText: Copyright (c) 2024 NVIDIA CORPORATION & AFFILIATES.
+# SPDX-FileCopyrightText: All rights reserved.
+# SPDX-License-Identifier: Apache-2.0
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from collections import OrderedDict
+from datetime import datetime, timezone
+from pathlib import Path
+
+import numpy as np
+import torch
+
+from earth2studio.models.auto import AutoModelMixin, Package
+from earth2studio.models.batch import batch_coords, batch_func
+from earth2studio.models.dx.base import DiagnosticModel
+from earth2studio.utils import (
+    check_extra_imports,
+    handshake_coords,
+    handshake_dim,
+)
+from earth2studio.utils.type import CoordSystem
+
+try:
+    from physicsnemo.utils.zenith_angle import cos_zenith_angle
+
+    from earth2studio.models.nn.afno_ssrd import SolarRadiationNet
+except ImportError:
+    SolarRadiationNet = None
+    cos_zenith_angle = None
+
+VARIABLES = [
+    "t2m",
+    "sp",
+    "tcwv",
+    "z50",
+    "z300",
+    "z500",
+    "z700",
+    "z850",
+    "z925",
+    "z1000",
+    "t50",
+    "t300",
+    "t500",
+    "t700",
+    "t850",
+    "t925",
+    "t1000",
+    "q50",
+    "q300",
+    "q500",
+    "q700",
+    "q850",
+    "q925",
+    "q1000",
+]
+
+
+@check_extra_imports("solarradiation-afno", [SolarRadiationNet, cos_zenith_angle])
+class SolarRadiationAFNO(torch.nn.Module, AutoModelMixin):
+    """Solar Radiation AFNO diagnostic model. Predicts the accumulated global surface solar
+    radiation over 6 hours [Jm^-2]. The model uses 31 variables as input and outputs
```

Comment:
todo: update the 6 hours doc string

Reply:
Hey  @albertocarpentieri, can you take a look at the doc strings for these models now and make sure they are right.

Reply:
@loliverhennigh I would call the variable Surface Solar Radiation Downwards (SSRD) as in ERA5. Why did you remove the southpole?

Reply:
Resolved